### PR TITLE
feat(editor): Migrate existing users to new canvas and set new canvas as default

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

- new users land on canvas v2 automatically
- old users get migrated to v2 automatically ONCE. If they switch back they stay on the old version
- both old and new users see the tooltip below for the first time

https://github.com/user-attachments/assets/73819f49-a6b3-4fb4-848f-09696c0e489f

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/N8N-7910/prepare-canvas-v2-for-open-beta
https://linear.app/n8n/issue/CAT-338/bug-new-canvas-callout-tooltip-not-correct-position-now

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/MainHeader/MainHeader.vue
Comment (r00gm) [inline packages/editor-ui/src/components/MainHeader/MainHeader.vue:1]: this change is great but i think it would be better to do it in a separate PR

**Review by r00gm — DISMISSED** [review]

**Review by cstuncsik — DISMISSED** [review]

**Review by r00gm — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.70.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.70.0)
